src/tactic/simps.lean

```diff
@@ -429,6 +429,9 @@ library_note "custom simps projection"
   * The option `trace` is set to `tt` when you write `@[simps?]`. In this case, the attribute will
     print all generated lemmas. It is almost the same as setting the option `trace.simps.verbose`,
     except that it doesn't print information about the found projections.
+  * if `add_additive` is `some nm` then `@[to_additive]` is added to the generated lemma. This
+    option is automatically set to `tt` when the original declaration was tagged with
+    `@[to_additive]`, where `nm` is the additive name of the original declaration.
```

Comment:
Do I recall correctly that this requires that the order of attributes is `@[to_additive, simps]` and not `@[simps, to_additive]`?
```suggestion
    `@[to_additive, simps]` (in that order), where `nm` is the additive name of the original declaration.
```

Reply:
Yes, that is correct. Your suggestion is indeed clearer.